docs/rc.xml.all

```diff
@@ -393,6 +393,18 @@
 
   </mouse>
 
+  <!--
+    Tablet buttons emulate regular mouse buttons.
```

Comment:
I meant this sentence as an introduction, more verbose it would be something like "All tablet buttons only do emulate the regular mouse buttons. That's why tablet buttons are mapped to mouse buttons". I'm not a native English speaker, though I would say that "Tablet buttons to emulate regular mouse buttons." does no longer work as a sentence. Could you please rephrase your suggestion or explain what meaning you had in mind?

Reply:
Let's leave it. It's brief and to the point :smile: